tests/driver_bme280/doc.txt

```diff
@@ -0,0 +1,7 @@
+/**
+* @defgroup      test_group_driver_bme280   Application test for bme280
+* @ingroup       test_group
```

Comment:
```suggestion
* @defgroup      tests_driver_bme280   Application test for bme280
* @ingroup       tests
```

Reply:
This was resolved in a past discussion , a new pr should be open and should contain all these changes